mmdet/models/anchor_heads/rpn_head.py

```diff
@@ -64,6 +64,7 @@ def get_bboxes_single(self,
         for idx in range(len(cls_scores)):
             rpn_cls_score = cls_scores[idx]
             rpn_bbox_pred = bbox_preds[idx]
+            anchors = mlvl_anchors[idx]
```

Comment:
how did it work before?

Reply:
It didn't. Probably it's been broken during merge. Moved this line to better align with the current master branch state.